apps/Box86/uninstall

```diff
@@ -12,6 +12,14 @@ cd ~/box86/build
 sudo make uninstall
 cd $HOME
 
+CODENAME=$(lsb_release -cs)
```

Comment:
no, thast not how `purge-installed` works (I made the same mistake once), use this instead:
```bash
if [ ! -z "$(file "$(readlink -f "/sbin/init")" | grep 64)" ]; then
    "${DIRECTORY}/purge-installed" "$(dirname "$0")" || exit 1
elif [ ! -z "$(file "$(readlink -f "/sbin/init")" | grep 32)" ]; then
    sleep 0.00001
else
    error "Can't detect OS architecture! something is very wrong!"
fi
```